installer/resources/pacbot_app/import_db.py

```diff
@@ -120,7 +120,7 @@ def get_provisioners(self):
         local_execs = [
             {
                 'local-exec': {
-                    'command': "mysql -u %s -p%s -h %s < %s" % (db_user_name, db_password, db_host, ReplaceSQLPlaceHolder.dest_file)
+                    'command': "mysql -u %s -p %s -h %s < %s" % (db_user_name, db_password, db_host, ReplaceSQLPlaceHolder.dest_file)
```

Comment:
This space will make problem as the command treat the text as DB name after the space. Please refer mysql man page,
https://linux.die.net/man/1/mysql


> The password to use when connecting to the server. If you use the short option form (-p), **you cannot have a space between the option and the password**. If you omit the password value following the --password or -p option on the command line, mysql prompts for one